src/Bindings/DeprecatedBindings.cpp

```diff
@@ -340,6 +340,49 @@ static int tolua_cWorld_SetSignLines(lua_State * tolua_S)
 
 
 
+template <typename T>
+int tolua_Vector3_Abs(lua_State * a_LuaState)
+{
+	// Retrieve the params, including self:
+	cLuaState L(a_LuaState);
+	Vector3<T> * self;
+	if (!L.GetStackValues(1, self))
+	{
+		tolua_error(a_LuaState, "invalid 'self' in function 'Vector3<T>:Abs'", nullptr);
+		return 0;
+	}
+
+	// Absolutize the vector:
+	self->Abs();
+	return 0;
+}
+
+
+
+
+
+template <typename T>
+int tolua_Vector3_Clamp(lua_State * a_LuaState)
+{
+	// Retrieve the params, including self:
+	cLuaState L(a_LuaState);
+	Vector3<T> * self;
+	T min, max;
+	if (!L.GetStackValues(1, self, min, max))
+	{
+		tolua_error(a_LuaState, "invalid 'self' in function 'Vector3<T>:Clamp'", nullptr);
```

Comment:
Copy-pasta error in the error message, any param could be wrong, not just `self`